patches/minecraft/net/minecraft/potion/Potion.java.patch

```diff
@@ -88,6 +88,17 @@
 +        ret.add(new net.minecraft.item.ItemStack(net.minecraft.init.Items.field_151117_aB));
 +        return ret;
 +    }
++    
++    /**
++     * Used as a last resort for determining {@code PotionEffect} sort order in GUIs.
++     * Defaults to the {@code PotionEffect}'s liquid color.
++     * @param potionEffect the {@code PotionEffect} instance containing the potion
++     * @return a value used to sort {@code PotionEffect}s in GUIs 
++     */
++    public int getGuiSortColor(PotionEffect potionEffect)
++    {
++        return this.field_76414_N;
```

Comment:
This should default to `getLiquidColor()` in case someone has overridden it for whatever reason.